src/LanguageProcessorEngine/Configuration/BaseLanguageProcessorConfiguration.php

```diff
@@ -7,14 +7,15 @@
 
 class BaseLanguageProcessorConfiguration extends BaseConfiguration implements LanguageProcessorConfigurationInterface
 {
-    const TYPE_KEY = 'type';
+    const TYPE = 'type';
     private string $type;
+    protected static array $requiredAttributes = [self::TYPE];
```

Comment:
Apologies this is my bad for not being specific enough, this wants to be the `$required` property from `BaseConfiguration`.

```suggestion
    protected static array $required = [self::TYPE];
```

Reply:
No worries, I also could be more attentive. Fixed.